docs/repo-docs/crafting-your-repository/configuring-tasks.mdx

```diff
@@ -326,30 +326,30 @@ Some tasks should always be ran no matter what, like a deployment script after a
 
 Some tasks can be ran in parallel despite being dependent on other packages. An example of tasks that fit this description are linters, since a linter doesn't need to wait for outputs in dependencies to run successfully.
```

Comment:
```suggestion
Some tasks can be ran in parallel despite being dependent on other packages. An example of tasks that fit this description are linters or type checkers, since a these tasks doesn't need to wait for outputs in dependencies to run successfully.
```

_Technically_, TypeScript as a type checker is a linter. But I can see how that would be confusing. Would this change be clearer?

Reply:
Yeah that works!

But I guess it feels like a more confusing example given my comment above:

> When NOT using Internal Packages, where the output of each package is `.d.ts` files, it would seem like you definetly can't start typechecking package A which depends on B, until B emits its `.d.ts` files

Reply:
Sorry, missed that. You're right, let's swap the example to being `lint`.